## 71% of end-end tests have passed
```
Total executed tests: 7
Failed tests: 2
Expected to fail tests: 0
Passed tests: 5
```
```
IDs of failed tests: 702742,703133 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=21117-more-Privacy-Policy-links&PR_ID=21119&APK_NAME=21119.apk&TR_CASE_IDS=702742,703133">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643938">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><p><blockquote><div>Device 1: Type `https://status.im` to `ChatMessageInput` </div><div>Device 1: Find `SendMessageButton` by `accessibility id`: `send-message-button` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:58: in test_community_copy_and_paste_message_in_chat_input
    self.channel.send_message(message)
../views/chat_view.py:993: in send_message
    self.send_message_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:84: in find_element
    raise exception
../views/base_element.py:77: in find_element
    return self.driver.find_element(self.by, self.locator)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6197fc728be94c74b2a5467712430771?auth=8c4fb19f842875f1a6d4a9477e0cfd8d#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643941">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:158: in test_restore_multiaccount_with_waku_backup_remove_switch
    self.home.reopen_app(sign_in=False)
../views/base_view.py:706: in reopen_app
    app_package = self.driver.current_package
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/extensions/android/common.py:74: in current_package
    return self.assert_extension_exists(ext_name).execute_script(ext_name)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: Could not retrieve the currently focused package and activity. Original error: Error executing adbExec. Original error: 'Command '/home/chef/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell dumpsys window displays' exited with code 1'; Command output: adb: device 'emulator-5554' not found
E   
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not retrieve the currently focused package and activity. Original error: Error executing adbExec. Original error: 'Command '/home/chef/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell dumpsys window displays' exited with code 1'; Command output: adb: device 'emulator-5554' not found
E   
E       at getResponseForW3CError (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/errors.js:1073:9)
E       at asyncHandler (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/protocol.js:491:57)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6197fc728be94c74b2a5467712430771?auth=8c4fb19f842875f1a6d4a9477e0cfd8d#73">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (5)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643944">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/681c0b82f11d448cbfa2174bfcf2a262?auth=d8e58fb0dbe8a64457519e7a94455552#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643943">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643942">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643940">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/670649e0b6434ab2b0480480dc57a378?auth=0c3b1b9a9de1a5357335af7ba178fbb1#164">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d45181119dda4b8a9648cdf1188e100c?auth=3be3a1c85c994fbf1f4c13d8d81217ad#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3643939">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1fb23ddfcadf4cde8dff0f71c1529ce6?auth=b4e58922611f90880c6a84b7760707d8#97">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9308ec11a024d708baa2107fe5e486c?auth=aab2a3de50f733582a711a3f93335067#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/common/metrics_confirmation_modal/style.cljs
+++ b/src/status_im/common/metrics_confirmation_modal/style.cljs
@@ -4,7 +4,7 @@
   {:margin-top        11
    :margin-horizontal 20
    :gap               21
-   :margin-bottom     15})
+   :margin-bottom     8})
 
 (def item-text
   {:margin-top  10
@@ -12,3 +12,9 @@
 
 (def info-text
   {:margin-top -5})
+
+(def privacy-policy
+  {:margin-top        4
+   :margin-horizontal 20
+   :margin-bottom     20
+   :align-items       :center})

--- a/src/status_im/common/metrics_confirmation_modal/view.cljs
+++ b/src/status_im/common/metrics_confirmation_modal/view.cljs
@@ -1,6 +1,7 @@
 (ns status-im.common.metrics-confirmation-modal.view
   (:require
     [quo.core :as quo]
+    [quo.foundations.colors :as colors]
     [quo.theme]
     [react-native.core :as rn]
     [status-im.common.metrics-confirmation-modal.style :as style]
@@ -52,14 +53,36 @@
   (toggle-metrics false)
   (hide-bottom-sheet))
 
+(declare view)
+
 (defn- on-privacy-policy-press
-  []
+  [settings?]
   (rf/dispatch
    [:show-bottom-sheet
-    {:content (fn []
-                [quo.theme/provider :dark
-                 [privacy/privacy-statement]])
-     :shell?  true}]))
+    {:content  (fn []
+                 [quo.theme/provider :dark
+                  [privacy/privacy-statement]])
+     :on-close (fn []
+                 (rf/dispatch [:show-bottom-sheet
+                               {:content (fn []
+                                           [quo.theme/provider :dark
+                                            [view {:settings? settings?}]])
+                                :shell?  true}]))
+     :shell?   true}]))
+
+(defn- privacy-policy-text
+  [settings?]
+  [rn/view {:style style/privacy-policy}
+   [quo/text
+    [quo/text
+     {:style {:color colors/white-opa-50}
+      :size  :paragraph-2}
+     (i18n/label :t/more-details-in-privacy-policy-1)]
+    [quo/text
+     {:size     :paragraph-2
+      :weight   :bold
+      :on-press #(on-privacy-policy-press settings?)}
+     (i18n/label :t/more-details-in-privacy-policy-2)]]])
 
 (defn view
   [{:keys [settings?]}]
@@ -79,22 +102,18 @@
       [quo/text
        {:size  :paragraph-2
         :style style/info-text}
-       (i18n/label :t/usage-data-shared-from-all-profiles)
-       (i18n/label :t/more-details-in-privacy-policy-1)
-       [quo/text
-        {:size     :paragraph-2
-         :weight   :bold
-         :on-press on-privacy-policy-press}
-        (i18n/label :t/more-details-in-privacy-policy-2)]]
+       (i18n/label :t/usage-data-shared-from-all-profiles)]
       [quo/text
        {:size  :paragraph-2
         :style style/info-text}
+       (i18n/label :t/usage-data-shared-from-all-profiles)
        (i18n/label :t/sharing-usage-data-can-be-turned-off)])]
    [quo/bottom-actions
     {:actions          :two-actions
      :blur?            true
      :button-one-label (i18n/label :t/share-usage-data)
      :button-one-props {:on-press on-share-usage}
-     :button-two-label (i18n/label (if settings? :t/do-not-share :t/not-now))
+     :button-two-label (i18n/label :t/not-now)
      :button-two-props {:type     :grey
-                        :on-press on-do-not-share}}]])
+                        :on-press on-do-not-share}}]
+   [privacy-policy-text settings?]])

--- a/src/status_im/contexts/onboarding/intro/style.cljs
+++ b/src/status_im/contexts/onboarding/intro/style.cljs
@@ -10,7 +10,7 @@
   {:flex           1
    :flex-direction :row
    :flex-wrap      :wrap
-   :align-self     :center})
+   :align-self     :flex-start})
 
 (def terms-privacy-container
   {:gap                8

